pkgs/development/python-modules/pdoc3/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildPythonPackage, fetchPypi
+, Mako
+, markdown
+, setuptools-git
+, setuptools_scm
+}:
+
+buildPythonPackage rec {
+  pname = "pdoc3";
+  version = "0.9.1";
```

Comment:
```suggestion
  version = "0.9.1";
  disabled = pythonOlder "3.5";
```

Reply:
Seems broken on 3.6 actually, so I just set it with pythonOlder 3.7